be/src/exec/pipeline/source_operator.h

```diff
@@ -79,8 +79,6 @@ class SourceOperator : public Operator {
 
     int64_t _last_scan_rows_num = 0;
     int64_t _last_scan_bytes = 0;
-
-    workgroup::WorkGroupPtr _workgroup = nullptr;
```

Comment:
why not reserve _workgroup in Source Operator. it seems JDBC/ES Source Operator will use workgroup later.

Reply:
They should inherit the `ScanOperator`.